rdkit/Chem/Draw/IPythonConsole.py

```diff
@@ -51,53 +41,57 @@
 
 ipython_useSVG = False
 ipython_3d = False
-molSize_3d = (450, 450)
-drawing_type_3d = "ball and stick"
-camera_type_3d = "perspective"
-shader_3d = "lambert"
-
+molSize_3d = (400, 400)
+drawing_type_3d = 'stick' # default drawing type for 3d structures
+bgcolor_3d = '0xeeeeee'
 # expose RDLogs to Python StdErr so they are shown
 #  in the IPythonConsole not the server logs.
 Chem.WrapLogs()
 
 
+def _addMolToView(mol,view,confId,drawAs):
+  if mol.GetNumAtoms()>=999 or drawAs == 'cartoon':
+    # py3DMol is happier with TER and MASTER records present
+    pdb = Chem.MolToPDBBlock(mol,flavor=0x20|0x10)
```

Comment:
Magic constants. Are there named constants for the two flavors? or use:
```
writeMasterRecord = 0x10
writeTerRecord = 0x20
pdb = Chem.MolToPDBBlock(mol, flavor=writeTerRecord|writeMasterRecord)
```
Better would be having these defined with the function.